README.md

```diff
@@ -1,2 +1,36 @@
-# trello-python-tests
+# Trello API Project
+This project allows to test Trello application using Behave.
 
+##Prerequisites
+For the correct functioning of the work environment, the following programs need to be installed:
+* Windows Environment: Server 2019 Standard.
+* Google Chrome version 79.0.3945.130
+
+##Installing
+To start setting up the environment with BDD (Behavior Driven Development).
+* Install [Python 3.8.1](https://www.python.org/downloads/)
+* Install [PyCharm 2019.3.2 (Professional Edition)](https://www.jetbrains.com/es-es/pycharm/download/#section=windows) 
+* Install [Git version 2.21.0.windows.1](https://git-scm.com/download/win)
+
+## Configuration
+This project was intended to work with python 3.8.1 A list of required libraries can be found on "requirements.txt" 
+file.
+* behave interpreter in PyCharm.
+* requests
+* jsonschema
+* Pytest to write unit tests.
+
+To finish this configuration we need add Pytest in Python Integrated Tool follow the next steps:
+
+Go File -> Settings -> Tools -> Python Integrated Tools and in Testing field in Default test runner select pytest option.  
+
+This means all required libraries can be installed from the project path using:
+```bash 
+pip install -r requirements.txt
+```
+## About the Project
+On the 'core' folder we intend to put generic steps and functions that can be used on any API testing. 
```

Comment:
Could you mention how to configure config.json

Reply:
done